CHANGELOG.md

```diff
@@ -2,7 +2,7 @@ UNRELEASED
 ----------
 
 * [ [#1516](https://github.com/digitalfabrik/integreat-cms/issues/1516) ] Fix save buttons alignment
-
+* [ [#1520](https://github.com/digitalfabrik/integreat-cms/issues/1520) ] Fix button name in side-by-side view
```

Comment:
```suggestion

* [ [#1520](https://github.com/digitalfabrik/integreat-cms/issues/1520) ] Fix button name in side-by-side view
```
Unfortunately, the changelog format is very strict - it needs exactly one line after the heading and two lines between the last bullet point and the next heading.